microdnf/context.cpp

```diff
@@ -337,22 +349,52 @@ void Context::load_rpm_repos(libdnf::repo::RepoQuery & repos, libdnf::repo::Repo
 // Single thread version.
 // TODO keep this and enable conditionally (compiletime or even runtime) or
 // drop when we know the multithreaded implementation is stable
-// void Context::load_rpm_repos(libdnf::repo::RepoQuery & repos, libdnf::repo::Repo::LoadFlags flags) {
-//     std::cout << "Updating repositories metadata and load them:" << std::endl;
-//     for (auto & repo : repos.get_data()) {
-//         try {
-//             load_rpm_repo(*repo.get());
-//             // std::cout << "Loading repository \"" << repo->get_config().name().get_value() << "\" into sack." << std::endl;
-//             repo->load(flags);
-//         } catch (const std::runtime_error & ex) {
-//             if (!repo->get_config().skip_if_unavailable().get_value()) {
-//                 std::cerr << "Error: Unable to load repository \"" << repo->get_id()
-//                           << "\" and \"skip_if_unavailable\" is disabled for it." << std::endl;
-//                 throw;
-//             }
-//         }
-//     }
-// }
+void Context::load_repos_single_thread(libdnf::repo::RepoQuery & repos, libdnf::repo::Repo::LoadFlags flags) {
```

Comment:
What are we going to do with this long-term? I'm about to move the `load_repos` to libdnf, I don't think we should have this on the API next to the multi-threaded version unless there's a reason for the duplicity?